src/app/home/home.component.css

```
@@ -0,0 +1,21 @@
+:host {
+  align-items: center;
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  background: #f1f1e7;
+}
+
+:host ::ng-deep .mat-progress-spinner circle,
+.mat-spinner circle {
+  stroke: #dae59b;
+}
+
+:host .mat-icon-button {
+  color: #dae59b;
+}
+
+:host .mat-card {
+  width: 75%;
+  background: #717f77;
+}
```

```suggestion
::ng-deep .mat-progress-spinner circle,
.mat-spinner circle {
  stroke: #dae59b;
}

.mat-icon-button {
  color: #dae59b;
}

.mat-card {
  background: #717f77;
  width: 75%;
}
```

two nits:
- You can leave of ng-host if you're not styling it directly
- alphabetize the css properties (google style nit)